src/AfterLogin/SearchPage.js

```diff
@@ -0,0 +1,247 @@
+            variant="contained"
+            className="copy-button"
+            onClick={() => handleCopy(deck.deckID)}
+          >
+            <div>Copy Deck</div>
+            <ContentCopyIcon />
+          </Button>
+        </AccordionDetails>
+      );
+      return (
+        <Accordion
+          expanded={expanded === deck.deckID}
+          key={`deck${deck.deckID}`}
+          onChange={() => handleExpanded(deck.deckID)}
+          className="search-list"
+        >
+          <AccordionSummary>
+            <Typography sx={{ width: "30%" }}>Deck Name:</Typography>
+            <Typography sx={{ width: "35%" }}>
+              <b>{deck.deckName}</b>
+            </Typography>
+            <Typography sx={{ width: "30%" }}>Deck Cards: </Typography>
+            <Typography sx={{ width: "5%" }}>
+              {deck.deckCards.length}
+            </Typography>
+          </AccordionSummary>
+          {cardList}
+        </Accordion>
+      );
+    });
+
+  const resultList =
+    result &&
+    result.map((deck) => {
+      const cardListBody = deck.cardInfos.map((card) => {
+        return (
+          <TableRow key={`card${card.cardID}`}>
+            <TableCell>{card.english}</TableCell>
+            <TableCell>{card.spanish}</TableCell>
+          </TableRow>
+        );
+      });
+      const cardList = (
+        <AccordionDetails>
+          <TableContainer component={Paper}>
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell>
+                    <b>English</b>
+                  </TableCell>
+                  <TableCell>
+                    <b>Spanish</b>
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>{cardListBody}</TableBody>
+            </Table>
+          </TableContainer>
+          <div>
+            <Button
+              variant="contained"
+              className="copy-button"
+              onClick={() => handleCopy(deck.deckID)}
+            >
+              <div>Copy Deck</div>
+              <ContentCopyIcon />
+            </Button>
+          </div>
+        </AccordionDetails>
+      );
+      return (
+        <Accordion
+          className="search-list"
+          expanded={expanded === deck.deckID}
+          key={`deck${deck.deckID}`}
+          onChange={() => handleExpanded(deck.deckID)}
+        >
+          <AccordionSummary>
+            <Typography sx={{ width: "30%" }}>Deck Name:</Typography>
+            <Typography sx={{ width: "35%" }}>
+              <b>{deck.deckName}</b>
+            </Typography>
+            <Typography sx={{ width: "30%" }}>Deck Cards: </Typography>
+            <Typography sx={{ width: "5%" }}>
+              {deck.deckCards.length}
+            </Typography>
+          </AccordionSummary>
+          {cardList}
+        </Accordion>
+      );
+    });
+
+  const resultDisplay = result && !!result.length && <div>{resultList}</div>;
```

Comment:
`!!result.length` is redundant as length is an integer. 0 is falsy and everything else is truthy. Thus `result.length` is the same as `!!result.length`